.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,166 @@
+name: Bug Report
+description: Broken or unintended behavior with one of our libraries.
+title: '[Bug] '
+labels: [bug-unconfirmed, question]
+body:
+- type: markdown
+  attributes:
+    value: |
+      ## Before Submitting your Issue
+      Please make sure that your question or issue is not already covered in [MSAL docs](https://learn.microsoft.com/en-us/entra/msal/dotnet/) or [samples](https://learn.microsoft.com/en-us/azure/active-directory/develop/sample-v2-code?tabs=apptype).
+
+      **Logs and network traces**
+      Without logs or traces, it is unlikely that the team can investigate your issue. Capturing logs and network traces is described in [Logging wiki](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/logging).
```

Comment:
Is there an equivalent learn.microsoft.com page for this? In another part of this template you link to https://learn.microsoft.com/en-us/azure/active-directory/develop/msal-logging-dotnet, which seems to mostly cover the same stuff

Reply:
Yes, @gladjohn - let's make sure that we point folks to Learn instead of the Wiki.

Reply:
updated @localden, please review